server/test/test-server.js

```diff
@@ -67,219 +70,222 @@ describe(`Test for "/signin" endpoint`, function() {
 	})
 })
 
-describe('DB controller unit tests', function() {
-	it('loadAll should return an array of doggo objects', done => {
+describe('DB controller unit tests', () => {
+	afterEach(() => {
+		sinon.restore()
+	})
+
+	it('loadAll should return an array of doggo objects', (done) => {
 		dbcontroller
 			.loadAll()
-			.then(res => {
+			.then((res) => {
 				res.should.be.a('array')
 				done()
 			})
-			.catch(err => {
+			.catch((err) => {
 				expect(err).to.be.a('null')
 			})
 	})
 
-	it('createAccount should create a user account', done => {
-		const formData = {
-			username: 'test1',
-			password: 'test1'
+	it('createAccount should create a user account', (done) => {
+		const objectStub = {
```

Comment:
probably give this a better name like `userStub`

I know it's just for a simple test, but we should keep it a good habit to always strive for good variable naming

Reply:
Done